src/components/error_message/ErrorMessage.styles.ts

```diff
@@ -0,0 +1,13 @@
+import styled from "styled-components"
+import { ERROR_COLOR, FONT_FAMILY } from "../../utils/Stylesheet"
+
+export const ErrorMessageStyle=styled.p`
```

Comment:
Ojo con el nombre, si bien son los estilos del parrafo, el nombre del componente debería ser lo que representa ese componente, en este caso un ErrorMessage y no un ErrorMessageStyle


```suggestion
export const ErrorMessage=styled.p`
```

Reply:
Le puse ese nombre porque entra en conflicto con el nombre del componente

Reply:
Quizás podríamos ponerle "ErrorText"?